example/googleMapsExample.js

```diff
@@ -0,0 +1,465 @@
+import { Ellipsoid, DebugTilesRenderer as TilesRenderer } from '../src/index.js';
+import {
+	Scene,
+	DirectionalLight,
+	AmbientLight,
+	WebGLRenderer,
+	PerspectiveCamera,
+	CameraHelper,
+	Vector3,
+	Quaternion,
+	Group,
+	sRGBEncoding,
+	Raycaster,
+	Vector2,
+	MeshPhongMaterial,
+	PlaneGeometry,
+	Plane,
+	PlaneHelper,
+	DoubleSide,
+	SphereGeometry,
+	MeshBasicMaterial,
+	Mesh,
+} from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+import { BufferGeometryUtils } from 'three/examples/jsm/utils/BufferGeometryUtils.js';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+import { DRACOLoader } from 'three/examples/jsm/loaders/DRACOLoader.js';
+import { GUI } from 'three/examples/jsm/libs/lil-gui.module.min.js';
+import Stats from 'three/examples/jsm/libs/stats.module.js';
+import { GlobeOrbitControls } from './GlobeOrbitControls.js';
+import { EllipsoidRegionHelper } from '../src/index.js';
+
+const apiOrigin = 'https://tile.googleapis.com';
+
+let camera, controls, scene, renderer, tiles, cameraHelper;
+let mainGroup;
+let statsContainer, stats;
+let ellipsoid = new Ellipsoid(), ellipsoidHelper, ellipsoidInitialised = false;
+let miscHelper;
+
+const raycaster = new Raycaster();
+const pointer = new Vector2();
+raycaster.firstHitOnly = true;
+
+let prevDist = 0;
+
+const params = {
+
+	'showGlobe': true,
+	'showGlobeHelper': false,
+	'showFrustumHelper': false,
+	'showMiscHelper': false,
+
+	'enableUpdate': true,
+	'enableCacheDisplay': false,
+	'enableRendererStats': false,
+
+	'apiKey': 'put-your-api-key-here',
+	'errorTarget': 6,
+	'loadSiblings': true,
+	'stopAtEmptyTiles': true,
+	'resolutionScale': 1.0,
+
+	'displayActiveTiles': false,
+	'displayBoxBounds': false,
+	'reload': reinstantiateTiles,
+
+};
+
+init();
+animate();
+
+function setupTiles() {
+
+	tiles.fetchOptions.mode = 'cors';
+
+	// Note the DRACO compression files need to be supplied via an explicit source.
+	// We use unpkg here but in practice should be provided by the application.
+	const dracoLoader = new DRACOLoader();
+	dracoLoader.setDecoderPath( 'https://unpkg.com/three@0.123.0/examples/js/libs/draco/gltf/' );
```

Comment:
For anyone trying to use this with the latest three.js this should be:

```js
dracoLoader.setDecoderPath( 'https://unpkg.com/three@0.152.0/examples/jsm/libs/draco/' );
```

Reply:
Yeah we should update the three.js version used for demos in the project.